storage/src/mmr/mem.rs

```diff
@@ -261,8 +260,7 @@ mod tests {
             0,
             "oldest_required_element should return 0 on empty MMR"
         );
-        let element = Bytes::from_static(b"01234567012345670123456701234567");
-        let element = <Sha256 as CHasher>::Digest::try_from(&element).unwrap();
+        let element = <Sha256 as CHasher>::Digest::from(*b"01234567012345670123456701234567");
```

Comment:
If we need to implement another `From<>`/`TryFrom<>` case for `&'static [u8]`, I think its worth it to avoid `*`.